vendor/brave-ios/tests/ledger_database_test.mm

```diff
@@ -1017,6 +1035,130 @@ - (void)testReservedAmount
   XCTAssertEqual(amount, 30.0);
 }
 
+#pragma mark - Publisher List / Info
+
+- (BATServerPublisherInfo *)serverPublisherInfo:(NSString *)publisherID
+{
+  return [self serverPublisherInfo:publisherID amounts:nil links:nil];
+}
+
+- (BATServerPublisherInfo *)serverPublisherInfo:(NSString *)publisherID
+                                        amounts:(nullable NSArray<NSNumber *> *)amounts
+                                          links:(nullable NSDictionary<NSString *, NSString *> *)links
+{
+  auto info = [[BATServerPublisherInfo alloc] init];
+  info.publisherKey = publisherID;
+  info.address = publisherID;
+  
+  const auto banner = [[BATPublisherBanner alloc] init];
+  banner.publisherKey = publisherID;
+  banner.desc = @"brave";
+  banner.amounts = amounts ?: @[ @1.0, @5.0, @10.0 ];
```

Comment:
This will always be the amount or?

Reply:
Its just the default amount we're setting for the test only. But yes on server side I believe this is also the default but they're unrelated